csharp-code-evaluator-ut/csharp-script-ut.csproj

```diff
@@ -36,7 +36,7 @@
   <ItemGroup>
     <Reference Include="nunit.framework, Version=2.6.3.13283, Culture=neutral, PublicKeyToken=96d09a1eb7f44a77, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\dependencies\nunit.framework.dll</HintPath>
+      <HintPath>C:\Program Files (x86)\NUnit 2.6.2\bin\framework\nunit.framework.dll</HintPath>
```

Comment:
@yyang-convey , why do you change to use hard coded version in c:\Program Files\.... ?
The repo is already configured to use version included in the project. You need to have git-lfs in your computer, and you will get the specific version of nunit.
Actually, we could deploy nunit-console exactly in the same way